examples/compute/externalvpngateway.yaml

```diff
@@ -129,6 +129,31 @@ spec:
 
 ---
 
+apiVersion: compute.gcp.upbound.io/v1beta1
+kind: RouterPeer
+metadata:
+  annotations:
+    meta.upbound.io/example-id: compute/v1beta1/externalvpngateway
+  labels:
+    testing.upbound.io/example-name: router1_peer2
+  name: router1-peer2
+spec:
+  forProvider:
+    advertisedRoutePriority: 100
+    interfaceSelector:
+      matchLabels:
+        testing.upbound.io/example-name: external-vpn-gateway-2
+    peerAsn: 64515
+    peerIpAddress: 169.254.1.2
+    regionSelector:
+      matchLabels:
+        testing.upbound.io/example-name: external-vpn-gateway
+    routerSelector:
+      matchLabels:
+        testing.upbound.io/example-name: external-vpn-gateway
+
+---
+
```

Comment:
Did you test the `examples/compute/externalvpngateway.yaml` example after adding the `RouterPeer` resource?

Reply:
The RouterPeer example was created in the generated example, but looks like it isn't being used by the other resources. Should it just be removed from the example?

Reply:
I think it would be good not to include an untested example.